Packs/QutteraWebsiteMalwareScanner/Integrations/QutteraWebsiteMalwareScanner/QutteraWebsiteMalwareScanner_test.py

```diff
@@ -0,0 +1,61 @@
+import json
```

Comment:
Please add a "given - when - then" documentation.
also, note that you have a few tests that fail.